index.js

```diff
@@ -23,30 +26,31 @@ module.exports = function(options) {
 
   var includeRegExp = new RegExp(prefix + 'include\\s*\\([^)]*["\'](.*?)["\'](,\\s*({[\\s\\S]*?})){0,1}\\s*\\)+');
 
-  function fileInclude(file) {
-    var self = this;
-
+  function fileInclude(file, enc, cb) {
     if (file.isNull()) {
-      self.emit('data', file);
-    } else if (file.isStream()) {
+      cb(null, file);
+    }
+
+    if (file.isStream()) {
```

Comment:
No need to save `this` in variable for clusure, shorter and callback is pretty node pattern